src/pages/Home.js

```diff
@@ -16,43 +20,63 @@ export default function Home() {
     localStorage.removeItem(`user`);
     navigate("/login");
   }
-
-  console.log(user);
+  
   return(
-    <> 
+    <div> 
       <h1>home</h1>
       <h5>not ready</h5>
-      <StyledSection>
-        <h3>{user}</h3>
-        <StyledDiv>
-          <p>test</p>
-          <button onClick={()=> {logout();}}>logout</button>
-        </StyledDiv>
-      </StyledSection>
-    </>
+      <StyledNavBar>
+        <Div1>
+          <StyledButton isActive={buttonBox1} onClick={()=> {setButtonBox2(false); setButtonBox3(false); setButtonBox1(buttonBox1? false: true)}}>{user}</StyledButton>
+          { buttonBox1 ? 
+              <StyledDiv3>
+                <button onClick={()=> logout()}>logout</button>
+              </StyledDiv3> : null}
+        </Div1>
+        <Div1>
+
+        </Div1>
+        <Div1>
+
+        </Div1>
+      </StyledNavBar>
+    </div>
   );
 }
 
-export const StyledSection = styled.section`
-  border-top: solid;
-  border-width: 1px;
-  display: grid;
-  column-gap: 50px;
-  grid-template-columns: auto auto auto;
-  
+export const Styldiv = styled.div`
```

Comment:
You apply an inconsistent naming here: Styldiv, Div1, StyledDiv3. Please use consistent and semantic naming.

Reply:
done